HoundCITest.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  HoundCITest.swift
+//  ArithmeticTools
+//
+//  Created by Brian Heim on 12/13/16.
+//  Copyright © 2016 James Bean. All rights reserved.
+//
+
+import Foundation
+
+class HoundCITest {
+    func testFunc() {
+        let testVar: [Int] = [3,4,5];
+        print(testVar);
```

Comment:
You can also just say: 

```Swift
let testVar = [3,4,5]
```

As the type (`Array<Int>`) is inferred by the compiler.